.github/workflows/api_ci.yml

```diff
@@ -49,6 +49,16 @@ jobs:
         export GO111MOUDULE=on
         sudo apt install golang-1.14-go
 
+    - name: generate json schema
+      working-directory: ./api
+      run: |
+        wget https://github.com/apache/apisix/archive/master.zip
+        mkdir ./build-tools/apisix/
+        unzip master.zip
+        sudo mv ./apisix-master/apisix/* ./build-tools/apisix/
+        rm -rf ./apisix-master
+        cd ./build-tools/ && lua schema-sync.lua > ../conf/schema.json
```

Comment:
I think we need to check if the `conf/schema.json` is the correct

Reply:
if `scheme.json` is not correct, `manager api` will not be able to run: 
https://github.com/apache/apisix-dashboard/pull/551/files#diff-662a29145b19ca440fb3bca6c8605c50R45-R48